### PR TITLE
[5.0.x] add comment to dependency. issues:#220

### PR DESCRIPTION
**Pull request by btfurukawatkr**
Please review for additional comment on #220 .

### CODE REVIEW

File: projectName-domain/pom.xml [hunk 1]
```diff
@@ -100,12 +100,14 @@
         <!-- == End Coding Support == -->
 
         <!-- == Begin MyBatis -->
```
Comment (ikeyat) [inline projectName-domain/pom.xml]: - [ ] Don't need this line. It is verbose.

**Comment by btfurukawatkr** [comment]
Please review again.
Added comments to mybatis-config.xml

File: infra/mybatis3/projectName-domain/src/main/resources/META-INF/mybatis/mybatis-config.xml [hunk 2]
```diff
@@ -9,13 +9,15 @@
...
-        <setting name="proxyFactory" value="CGLIB"/>
 <!--
         <setting name="defaultExecutorType" value="REUSE" />
         <setting name="jdbcTypeForNull" value="NULL" />
-        <setting name="proxyFactory" value="JAVASSIST" />
         <setting name="localCacheScope" value="STATEMENT" />
 -->
+<!-- Defaul value for proxyFactory has chenged to JAVASSIST since mybatis 3.3.0 .
+     To use CGLIB as a proxyFactory, change value to "CGLIB" and add dependency for CGLIB in pom.xml
+        <setting name="proxyFactory" value="JAVASSIST" />
```
Comment (btshimizukza) [inline infra/mybatis3/projectName-domain/src/main/resources/META-INF/mybatis/mybatis-config.xml]: please change to `CGLIB`

Comment (btfurukawatkr) [inline infra/mybatis3/projectName-domain/src/main/resources/META-INF/mybatis/mybatis-config.xml]: changed to CGLIB and fixed comment.

**Comment by btshimizukza** [comment]
@ikeyat 

OK.

**Pull request merged**